rpcd/playbooks/roles/rpc_maas/templates/galera_check.yaml.j2

```diff
@@ -34,3 +34,65 @@ alarms      :
             if (percentage(metric["mysql_current_connections"], metric["mysql_max_configured_connections"]) > {{ mysql_connection_warning_threshold }} ) {
                 return new AlarmStatus(WARNING, "Mysql connectons has exceeded {{ mysql_connection_warning_threshold }}% of configured maximum");
             }
+    mysql_max_used_connections :
+        label                   : mysql_max_used_connections--{{ ansible_hostname }}
+        notification_plan_id    : "{{ maas_notification_plan }}"
+        criteria                : |
+            :set consecutiveCount={{ maas_alarm_local_consecutive_count }}
+            if (metric["mysql_max_seen_connections"] > metric["mysql_max_configured_connections"]) {
```

Comment:
Thanks @andymcc  for the input. I was going off what Bjoern said in one of the bug report comments:
"Basically current number of sessions , max_used_connections < max_connections limit".
According to the documentation:
Max_used_connections -  The maximum number of connections that have been in use simultaneously since the server started. 
max_connections - The maximum permitted number of simultaneous client connections

Also in the documentation it mentions the possiblity of a "Too many connections error"[1], which I presume is what we are trying to monitor, right @BjoernT?

[1] http://dev.mysql.com/doc/refman/5.7/en/too-many-connections.html

Thoughts?